Sources/SwiftSentry/Utilities/StowedExceptions.swift

```diff
@@ -0,0 +1,30 @@
+#if os(Windows)
+import WinSDK
+
+// From https://learn.microsoft.com/en-us/windows/win32/wer/stowed-exception-information-v2
+struct STOWED_EXCEPTION_INFORMATION_V2 {
```

Comment:
why can't we just use the struct in `WinSDK`?

Reply:
It's not exposed in the WinSDK, see https://learn.microsoft.com/en-us/windows/win32/wer/stowed-exception-information-v2#remarks